sunpy/net/dataretriever/client.py

```diff
@@ -225,7 +107,34 @@ def _can_handle_query(cls, *query):
         `sunpy.net.fido_factory.UnifiedDownloaderFactory`
         class uses to dispatch queries to this Client.
         """
-        raise NotImplementedError
+        from sunpy.net import attrs as a
+
+        required = {a.Time, a.Instrument}
+        adict = cls.register_values()
+        optional = {i for i in adict.keys()}
```

Comment:
This needs to be everything in register_values that isn't in required?
```suggestion
        optional = {i for i in adict.keys()}difference(required)
```

Reply:
yeah, True will change it accordingly